### ISSUE TITLE
Track touch for swipe/pan gestures

### ISSUE THREAD

**Issue by dlee** [layerJS/layerJS#73]
Right now, sliding frames only move after a swipe gesture has been completed. It would be good if the frames actually tracked the touch to make things feel more natural.

**Comment by thomashandorf** [comment]
Good point. For this we need to parametrise the  transitions which is a bit of refactoring. Is on the wishlist. Thanks!
